README.md

```diff
@@ -9,7 +9,17 @@ We'd love to see you contributing to our projects, please just follow a few conv
 - No trailing whitespace. Blank lines should not have any spaces.
 - Do not indent after private/protected.
 - Use Ruby >= 1.9 syntax for hashes. Prefer `{ a: :b }` over `{ :a => :b }`.
-- Prefer `&&`/`||` over and/or.
+- Prefer `&&`/`||` over `and`/`or`.
+- Use `and`/`or` only for the early returns:
+``` ruby
+def some_method_with_and
+  head 401 and return
```

Comment:
@Mandaryn Does it need to be complex? This is only for illustrating the idea of the usage, not the context.